rest-api/flow/docs/grpc-api.md

```
@@ -1356,124 +1676,632 @@ every Task is returned subject to pagination.
 | Field | Type | Label | Description |
 | ----- | ---- | ----- | ----------- |
-| offset | [int32](#int32) |  |  |
-| limit | [int32](#int32) |  |  |
+| id | [UUID](#v1-UUID) |  |  |
+| name | [string](#string) |  |  |
+| description | [string](#string) |  |  |
+| operation_type | [OperationType](#v1-OperationType) |  |  |
+| operation_code | [string](#string) |  | Specific operation code (e.g., &#34;power_on&#34;, &#34;upgrade&#34;) |
+| rule_definition_json | [string](#string) |  | JSON-encoded RuleDefinition |
+| is_default | [bool](#bool) |  |  |
+| created_at | [google.protobuf.Timestamp](#google-protobuf-Timestamp) |  |  |
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)